### PR TITLE
Create Anirudh Anup

### LINKED ISSUE #173: Add qwe123pro  to the list
I am Anirudh Anup, I would like to add my profile to linkfree

### PR DESCRIPTION
**Pull request by qwe123coder**
Fixes #173 

## Proposed Changes

  -
  -
  -

## Screenshots

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/175"></a>

### CODE REVIEW

**Review by FayasNoushad — APPROVED** [review]

File: public/Anirudh Anup [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Anirudh Anup",
+  "bio": "I am Kaiwalya Koparkar, A passionate 13 year old developer from India, I am always open to collaborating on projects",
```
Comment (voidz7) [inline public/Anirudh Anup:3]: Correct your name 🦈

**Comment by qwe123coder** [comment]
@FayasNoushad a small note pls do not just approve all PRs gust like that pls check for  grammar errors, missing a comma , missing of brackets etc

**Review by voidz7 — COMMENTED** [review]
looks good ⚡

**Comment by kaiwalyakoparkar** [comment]
@FayasNoushad Please go through the PR thoroughly before approving or adding comments. We really appreciate your contribution :slightly_smiling_face:

File: public/Anirudh Anup [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Anirudh Anup",
+  "bio": "I am Anirudh Anup, A passionate 13 year old developer from India, I am always open to collaborating on projects",
+  "avatar": "https://github.com/kaiwalyakoparkar.png",
```
Comment (rohan-kulkarni-25) [inline public/Anirudh Anup:4]: - Please check the Avatar url I seems not to be yours.

**Review by rohan-kulkarni-25 — CHANGES_REQUESTED** [review]
Please c

File: public/Anirudh Anup (hunk 2 above)
Comment (qwe123coder) [reply public/Anirudh Anup:4]: opps thanks

File: public/Anirudh Anup [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "http://github.com/qwe123coder",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on polywork",
+      "url": "https://www.polywork.com/anirudh_anup",
+      "icon": "polywork"
```
Comment (eddiejaoude) [inline public/Anirudh Anup:14]: I don't think polywork is support yet, so this will get filter out for now and not display

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
The file name is not valid, needs to have no spaces and be in `public/data` folder. Are the docs not clear, do they need updating?

File: public/Anirudh Anup (hunk 3 above)
Comment (qwe123coder) [reply public/Anirudh Anup:14]: oh ok, so should I remove it?

**Comment by qwe123coder** [comment]
> The file name is not valid, needs to have no spaces and be in `public/data` folder. Are the docs not clear, do they need updating?

oof its my bad. my pc is  and GitHub are acting weird lately

**Comment by qwe123coder** [comment]
i am not able to move my data form ```public/ ``` to ```public/data``` 
I will make a new or

**Pull request closed without merging**